docs/current/sdk/nodejs/snippets/get-started/step4/build.js

```diff
@@ -11,7 +11,7 @@ connect(async (client) => {
 
   // mount cloned repository into Node image
   const runner = client
-    .container(node)
+    .container({id: node})
```

Comment:
How about using node directly instead of `client.container({id: node})`?

Reply:
Because container has only optional arguments:
```ts
container(opts?: ClientContainerOpts)
```